layers/+completion/ivy/README.org

```diff
@@ -0,0 +1,56 @@
+#+TITLE: Ivy layer
+
+* Table of Contents                                         :TOC_4_gh:noexport:
+ - [[#description][Description]]
+ - [[#install][Install]]
+ - [[#configuration][Configuration]]
+ - [[#key-bindings][Key Bindings]]
+   - [[#transient-state][Transient state]]
+
+* Description
+This layer enables Ivy for completion. It will replace the default completion by [[https://github.com/emacs-helm/helm][Helm]].
+
+These completion systems are the central control towers of Spacemacs, they are
+used to manage buffers, projects, search results, configuration layers, toggles
+and more...
+
+Mastering your choice of completion system will make you a Spacemacs power user.
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =ivy= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+* Configuration
+You can customize ivy with the following variables:
+
+1. ivy-wrap Whether C-n and C-p should wrap-around when at the the first or last candidate. The default value is nil.
```

Comment:
This list looks a little weird, maybe wrap the variable names inside equal signs: `=ivy-wrap=` to format them or matbe turn this list into a table, whatever you feel looks best.